book/src/integrations/pam_and_nsswitch/suse.md

```diff
@@ -16,16 +16,28 @@ authentication:
 > copy the `-pc` files. You can then edit the files safely.
 
 ```bash
+rm /etc/pam.d/common-account
```

Comment:
We *really* need a `YOU MUST DO THIS AS ROOT NOT JUST WITH SUDO` warning on this one, since that'll instantly break things 😱